src/scss/components/file/_index.scss

```diff
@@ -0,0 +1,8 @@
+@import "config";
+
+/// ========== PARTIALS ========== ///
+@import "./hx-file-control";
+@import "./hx_drop-fence";
+@import "./hx_drop-zone";
```

Comment:
Please use hyphen instead of underscore.

_NOTE: this was probably my fault, because I misspelled using underscores in the code snippet of a previous comment._

Reply:
Partials filenames changed to used underscore instead of hyphen.